transformer/events_creator_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/elastic/cloudbeat/dataprovider/types"
 	"github.com/elastic/cloudbeat/evaluator"
 	"github.com/elastic/cloudbeat/resources/fetching"
-	"github.com/elastic/cloudbeat/resources/fetching/fetchers"
+	fetchers "github.com/elastic/cloudbeat/resources/fetching/fetchers/k8s"
```

Comment:
The failure with golangci-lint happens because you need to update the exclude rules in https://github.com/elastic/cloudbeat/blob/a89b3eef8aab022324a0b7ce67971338c1aedefc/.golangci.yaml#L40-L41